setup.py

```diff
@@ -27,7 +27,8 @@
     ],
     entry_points={
         "console_scripts": [
-            'conda-mirror = conda_mirror.conda_mirror:cli'
+            'conda-mirror = conda_mirror.conda_mirror:cli',
+            'conda-diff-tar = conda_mirror.diff_tar:main',
```

Comment:
Shouldn't this be a subcommand of conda-mirror, rather than a separate command?

Reply:
Not sure, I realized that just adding the new functionality as options wouldn't make too much sense.  Having a sub command it of course possible too, for example `conda-mirror diff-tar`.

Reply:
@xhochy @mariusvniekerk - any opinions?